### PR TITLE
[FIX] the website uses its own qweb model by now

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
LGTM :+1: 

(ref: https://github.com/odoo/odoo/commit/510a4f51d2a5b75f7b4c3b8b345f693eb044b337)

File: website_backend_views/model/ir_qweb.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 
 class IrQweb(models.Model):
```
Comment (yvaucher) [inline website_backend_views/model/ir_qweb.py]: Don't you want to rename the model as well?

Comment (hbrunn) [inline website_backend_views/model/ir_qweb.py]: @yvaucher I was tempted to quote Shakespeare on this one, but you're right of course. As a followup, the file name and the import have to change too

**Comment by yvaucher** [comment]
@hbrunn Thanks :+1:

File: website_backend_views/model/ir_qweb.py (hunk 1 above)
Comment (yvaucher) [inline website_backend_views/model/ir_qweb.py]: @hbrunn I prefer to quote Einstein about simplicity. Though his sentence has less soul.

**Pull request merged**